etc/packaging/appimages/cartographer-module-x86_64.yml

```diff
@@ -0,0 +1,72 @@
+# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
```

Comment:
Could you please post a diff of the carto grpc yaml like I did here?
https://github.com/viamrobotics/viam-orb-slam3/pull/38#discussion_r1165925671

Reply:
Copy/pasted [`/etc/package/appimages/carto_grpc_server-aarch64.yml`](https://github.com/viamrobotics/viam-cartographer/blob/e543ccc47d7ed10094a26d0a4ea5f88dbfeb1e1c/etc/packaging/appimages/carto_grpc_server-x86_64.yml) and slightly modified:

```bash
diff etc/packaging/appimages/carto_grpc_server-x86_64.yml etc/packaging/appimages/cartog
rapher-module-x86_64.yml
7a8
>  - cp ../../../bin/cartographer-module $TARGET_APPDIR/usr/bin/
22,23c23,24
<     id: com.viam.carto_grpc_server
<     name: carto_grpc_server
---
>     id: com.viam.cartographer-module
>     name: cartographer-module
68c69
<         AIX_TARGET: usr/bin/carto_grpc_server
---
>         AIX_TARGET: usr/bin/cartographer-module
71c72
<   update-information: zsync|http://packages.viam.com/apps/slam-servers/carto_grpc_server-${BUILD_CHANNEL}-x86_64.AppImage.zsync
---
>   update-information: zsync|http://packages.viam.com/apps/slam-servers/cartographer-module-${BUILD_CHANNEL}-x86_64.AppImage.zsync
```